.travis.yml

```diff
@@ -36,15 +36,16 @@ matrix:
       if: (branch = master OR branch = v3) AND (NOT type in (pull_request))
 
 before_install:
+  # Remove oclint as it conflicts with GCC (indirect dependency of hdf5)
```

Comment:
How about moving this comment to before `brew cask uninstall oclint`?
Also, put a blank line after that line to make the scope of this comment clear.

Reply:
Actually I first tried that but it was not valid YAML syntax... (maybe because it is a part of a list)

https://github.com/kmaehashi/chainer/commit/53cee9b91995620fff8b4b85da4bfb2b85141c91
https://travis-ci.org/kmaehashi/chainer/builds/308836583

Reply:
Oh, I see!
That's fine, then.